Mage.Sets/src/mage/cards/f/FaerieFencing.java

```diff
@@ -0,0 +1,116 @@
+package mage.cards.f;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+import mage.MageObject;
+import mage.MageObjectReference;
+import mage.abilities.Ability;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.WatcherScope;
+import mage.filter.FilterPermanent;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.stack.Spell;
+import mage.target.common.TargetCreaturePermanent;
+import mage.watchers.Watcher;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class FaerieFencing extends CardImpl {
+
+    public FaerieFencing(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{X}{B}");
+        
+        // Target creature gets -X/-X until end of turn. It gets an additional -3/-3 if you controlled a Faerie as you cast this spell.
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent());
+        this.getSpellAbility().addEffect(new FaerieFencingEffect());
+        this.getSpellAbility().addWatcher(new ControlledFaerieAsSpellCastWatcher());
+    }
+
+    private FaerieFencing(final FaerieFencing card) {
+        super(card);
+    }
+
+    @Override
+    public FaerieFencing copy() {
+        return new FaerieFencing(this);
+    }
+}
+
+class FaerieFencingEffect extends OneShotEffect {
+
+    public FaerieFencingEffect() {
+        super(Outcome.UnboostCreature);
+        this.staticText = "target creature gets -X/-X until end of turn. It gets an additional -3/-3 if you controlled a Faerie as you cast this spell.";
+    }
+
+    protected FaerieFencingEffect(final FaerieFencingEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        ControlledFaerieAsSpellCastWatcher watcher = game.getState().getWatcher(ControlledFaerieAsSpellCastWatcher.class);
+        MageObject mo = source.getSourceObject(game);
+
+        if (watcher == null || mo == null) {
```

Comment:
This shouldn't end up mattering as presumably these objects aren't null in actual usage, but since they're only accessed for the additional effect, I'd check them in the reduction adjustment conditional rather than returning false here